README.md

```diff
@@ -195,6 +201,63 @@ try {
 }
 ```
 
+<a name="scav" />
+
+### Add custom request attributes
+
+You can use the SDK to add custom request attributes to current traced service. Custom request attributes allow you to do easier/better filtering of your requests in Dynatrace.
+
+Adding custom request attributes to the currently traced service call is pretty simple. Just call one of the addCustomRequestAttribute methods with your key and value:
+
+```Java
+oneAgentSDK.addCustomRequestAttribute("region", "EMEA");
+oneAgentSDK.addCustomRequestAttribute("salesAmount", 2500);
+```
+
+When no service call is being traced, the custom request attributes are dropped.
+
+<a name="webrequests"/>
+
+## Trace incoming web requests
+
+You can use the SDK to trace incoming web requests. This might be useful if Dynatrace does not support the respective web server framework or language.
+
+To trace any kind of incoming web request you first need to create a WebServerInfo object. The info object represents the endpoint of your web server (webservername, applicationname and context root). This object should be reused for all traced webrequests for the same application.
```

Comment:
I suggest to simplify "to trace any kind of incoming web request" => "to trace an incoming web request"

Reply:
webservername => web server name
applicationname => application name

Reply:
"for all traced webrequests for the same application" => "for all traced **web requests within** for the same application"